exercises/square-root/introduction.md

```diff
@@ -0,0 +1,10 @@
+# Introduction
+
+We are launching a deep space exploration rocket and we need a way to make sure the navigation system stays on target.
+
+As the first step in our calculation, we take a target number and find its square root (that is, the number that when multiplied by itself equals the target number).
+
+The journey will be very long.
+To make the batteries last as long as possible, we had to make our rocket's onboard computer very power efficient.
+Unfortunately that means that we can't rely on fancy math libraries and functions, as they use more power.
+Instead we want to calculate the square root directly.
```

Comment:
Note, "directly" suggests that the library functions do something less direct than the suggested algorithms. Using the library functions is just as direct implementing this yourself. 

We may want to consider replacing "directly" with something more accurate, like suggesting that we want to build a minimal math library that has only the one function we actually need: square root. 

Or we can leave it as is for now.